pkg/admissioncontroller/webhook/admission/admissionpluginsecret/add.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2023 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
```

Comment:
Please add docs about this in https://github.com/gardener/gardener/blob/master/docs/concepts/admission-controller.md

Reply:
Added in 52b8aa7b1f0fb061cf83871c453cc6974442b851.